### PR TITLE
21432-Implement-TestCoverage-using-MetaLink-

### PR DESCRIPTION
**Pull request by MarcusDenker**
Implement TestCoverage using MetaLink
	https://pharo.fogbugz.com/f/cases/21432/Implement-TestCoverage-using-MetaLink

Method level for now

### CODE REVIEW

**Comment by MarcusDenker** [comment]
not related: testByteEncoderFromUrl – Windows32.Zinc.Character.Encoding.Tests.ZnCharacterEncoderTests

File: src/SUnit-UI/TestRunner.class.st [hunk 1]
```diff
@@ -386,23 +386,29 @@ TestRunner >> classesSearchFrame [
...
 		ifTrue:  [ UIManager default inform: 'Congratulations. Your tests cover all code under analysis.' ]
 		ifFalse: [ 
 			Smalltalk tools messageList 
-				browse: wrappers
-				title: 'Not Covered Code (' , (100 - (100 * wrappers size // methods size)) printString , '% Code Coverage)'].
+				browse: notExecuted
+				title: 'Not Covered Code (' , (100 - (100 * notExecuted size // methods size)) printString , '% Code Coverage)'].
+	
+	"cleanup the annotations"
+	methods do: [ :each | each ast removeProperty: #tagExecuted ifAbsent: [ ] ].
```
Comment (gcotelli) [inline src/SUnit-UI/TestRunner.class.st:410]: Why not putting this cleanup into an ensure: ?

**Comment by astares** [comment]
Would be nice in the future to visualize the coverage in the UI after running the tests
(see screenshot on http://www.eclemma.org)

File: src/SUnit-UI/TestRunner.class.st (hunk 1 above)
Comment (MarcusDenker) [reply src/SUnit-UI/TestRunner.class.st:410]: yes! There is HDTestCoverage, too. I will make another pass...

**Comment by MarcusDenker** [comment]
Yes, and we need to go down to the basic block , line / satement level...

**Pull request merged**